CHANGELOG.md

```diff
@@ -1,5 +1,115 @@
 # Release Notes
 
+## Version 1.30.0 - 2023-03-31
+
+Features:
+* Implement experimental QEMU on Windows [#15781](https://github.com/kubernetes/minikube/pull/15781)
+
+Major Improvements:
+* Ensure only one `minikube tunnel` instance runs at a time [#15834](https://github.com/kubernetes/minikube/pull/15834)
+* Infer HyperKit HostIP as Gateway rather than hardcode to 192.168.64.1  [#15720](https://github.com/kubernetes/minikube/pull/15720)
+* multi-node: Add support for volumes using CSI addon [#15829](https://github.com/kubernetes/minikube/pull/15829)
+
+Minor Improvements:
+* QEMU: Rename `user` network to `builtin` and update documentation [#15793](https://github.com/kubernetes/minikube/pull/15793)
+* none driver: Look for cri-dockerd instead of hardcoding [#15784](https://github.com/kubernetes/minikube/pull/15784)
+* Replace instances of `k8s.gcr.io` with `registry.k8s.io` [#16200](https://github.com/kubernetes/minikube/pull/16200)
+* Handle CRI config of NetworkPlugin and PauseImage [#14703](https://github.com/kubernetes/minikube/pull/14703)
+* Remove deprecated `container-runtime` flag from Kubernetes v1.24+ [#16124](https://github.com/kubernetes/minikube/pull/16124)
+* Add cri-dockerd logs to `minikube logs` output [#16149](https://github.com/kubernetes/minikube/pull/16149)
+* Add ingress logs to `minikube logs` output [#15775](https://github.com/kubernetes/minikube/pull/15775)
+* Add default cni logs to `minikbue logs` output [#15909](https://github.com/kubernetes/minikube/pull/15909)
+* Add JSON output option to `miniikube service list` [#15831](https://github.com/kubernetes/minikube/pull/15831)
+* Add kicbase download process to JSON output [#15685](https://github.com/kubernetes/minikube/pull/15685)
+* Implement `--docs` for `minikube addons list -o json` [#15866](https://github.com/kubernetes/minikube/pull/15866)
+* Implement `--skip-audit` flag and skip adding `profile` commands to audit log [#15872](https://github.com/kubernetes/minikube/pull/15872)
+* Implement `--last-start-only` flag to `minikube logs` to only show last start logs  [#15770](https://github.com/kubernetes/minikube/pull/15770)
+
+Bugs:
+* Fix failing to enable metallb addon [#16056](https://github.com/kubernetes/minikube/pull/16056)
```

Comment:
nit: but for all addons we could do

addon metallb: fix enalbing 
addon gcp-auth: fix ...

but this is okay too just would be easier for the reader to skip irrelevent ones if they dont care about some addon

Reply:
Done